planner/core/stats.go

```diff
@@ -327,6 +328,9 @@ func (ds *DataSource) DeriveStats(childStats []*property.StatsInfo, selfSchema *
 			}
 		}
 	}
+	if len(ds.indexMergeHints) > 0 {
```

Comment:
Do we need consider situation when there is no hint. I mean there will be no log if binding doesn't work.